lib/aerosol/launch_configuration.rb

```diff
@@ -90,6 +92,12 @@ def to_s
 }}
   end
 
+  def corrected_user_data
+    realized_user_data = user_data.is_a?(Proc) ? user_data.call : user_data
```

Comment:
Might be nice to pass the `meta_data` to the `user_data` proc, since the argument could just be ignored.